test/tree/imagesTree.test.ts

```diff
@@ -11,77 +11,92 @@ const testImages: DockerImage[] = [
     {
         Name: 'a',
         Id: 'sha256:b0648d86f18e6141a8bfa98d4d17d5180aa2699af7f27eac5491fd1f950f6f05',
-        CreatedTime: generateCreatedTimeInSec(2)
+        CreatedTime: generateCreatedTimeInSec(2),
+        Size: 2 * 1024 * 1024,
```

Comment:
Consider changing the test data so that the image sizes are not increasing throughout (and the data is not already sorted) 😁